iocage/lib/ioc_start.py

```diff
@@ -388,6 +384,33 @@ def start_network_vnet(self, nic, bridge, mtu, iface, ip, defaultgw, jid):
             checkoutput(["ifconfig", bridge, "addm", f"{nic}:{jid}", "up"],
                         stderr=STDOUT)
             checkoutput(["ifconfig", f"{nic}:{jid}", "up"], stderr=STDOUT)
+        except CalledProcessError as err:
+            return f"{err.output.decode('utf-8')}".rstrip()
+        else:
+            return
+
+    def start_network_vnet_addr(self, nic, bridge, mtu, iface, ip, defaultgw, jid):
```

Comment:
Some variables are not used, they should be removed from the definition and all calls.